lib/migrations/2020-01-15.ts

```diff
@@ -348,6 +353,29 @@ export class Migration20200115 extends Migration {
       rawStructure.nonwrapped![StorageKey.BiometricsState] = biometricPrefs.enabled;
       rawStructure.nonwrapped![StorageKey.MobileBiometricsTiming] = biometricPrefs.timing;
     }
+    const lastExportDate = await this.services.deviceInterface.getRawStorageValue(
+      LegacyKeys.MobileLastExportDate
+    );
+    const doNotWarnUnsupportedEditors = await this.services.deviceInterface.getRawStorageValue(
+      LegacyKeys.MobileDoNotWarnUnsupportedEditors
+    );
+    const legacyOptionsState = await this.services.deviceInterface.getJsonParsedRawStorageValue(LegacyKeys.MobileOptionsState);
+    let migratedOptionsState = {}
+    if (legacyOptionsState) {
+      const legacySortBy = legacyOptionsState.sortBy;
+      migratedOptionsState = {
+        sortBy: isNullOrUndefined(legacySortBy) || legacySortBy === 'updated_at' || legacySortBy === 'client_updated_at' ? CollectionSort.UpdatedAt : legacySortBy,
+        sortReverse: legacyOptionsState.sortReverse ?? false,
+        hideNotePreview: legacyOptionsState.hidePreviews ?? false,
+        hideDate: legacyOptionsState.hideDates ?? false
+      }
+    }
+    rawStructure.nonwrapped![StorageKey.MobilePreferences] = {
```

Comment:
Nonwrapped? Should be part of wrapped. Nonwrapped is only information needed pre-unlock. In this case all this info appears to be post-unlock relevant.